README.md

```diff
@@ -1,3 +1,14 @@
 ## CoffeeOps Starter Kit
 
 So you want to start your own CoffeeOps?
+
+When first starting out, you'll probably have just a few folks (2-3). So, let everyone decide on the style of the meeting. Once, you have 7 or more folks, go ahead with the lean coffee approach for the meeting (please see http://leancoffee.org/ for more info).
+
+Here are your key factors for success:
+
+1. Consistency
+2. Availability and accessibility of the chosen venue
+  a) For example, is it easy to get to for the people you are targeting. Check for steps/elevator availability
+3. Provide a few ice breaker ideas to get folks talking
```

Comment:
For this one, I was thinking we would provide a few ice breaker ideas. Will go ahead and merge but would be good to pull this out as a separate section and expand with ideas :)